datasets/gov/vat/r2_fitosanitarinis_registras.csv

```diff
@@ -0,0 +1,17 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vat/fitosanitarinis_registras,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Subjektas,,,id,,,4,open,,"Pateikiama informacija apie ūkio subjektus, įregistruotus Fitosanitariniame registe (ūkio subjekto atpažinties kodas; pavadinimas; veiklos vykdymo adresas; veiklos ir produkcijos rūšis)",
+,,,,,id,,,,,4,open,,Įrašo identifikatorius,
+,,,,,subjektas,string,,,,2,open,,Subjekto pavadinimas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"", type: ""ref"", ref: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"")	",4,open,"spinta:205, manifest:1290",,
+,,,,,juridinis,boolean,,,,4,open,,Ūkio subjekto tipas,
+,,,,,tel_nr,string,,,,1,open,,Telefono numeris,
```

Comment:
Mano nuomone, jeigu duomenys bus pateikti taip kaip dabar, tai net įgyvendinimus `spinta:161`, brandos lygis nebus 4, nes telefono numeris nurodytas nestandartiniu formatu.
Pirmiausia reikėtų sutvarkyti duomenis. 
![image](https://user-images.githubusercontent.com/85924252/231431148-5b33d200-cd77-487a-b95b-35122a482620.png)

Atsižvelgiant į tai, rekomenduočiau pakartotinai įvertinti, ar šis komentaras būtinas ir ar jo pakanka.

Reply:
Gerai sakote. Telefono numeriai išties pateikti nestandartiniu formatu. Komentarą įdėjau tik dėl to, jog sutraukiau to paties subjekto kelius numerius į vieną, todėl kaip ir gaunasi Array tipas. Bet šiuo atveju geriau pašalinsiu komentarą, kaip ir siūlote.